netty/src/main/java/io/grpc/netty/InternalHandlerSettings.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2016, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.netty;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.base.Preconditions;
+import io.grpc.Internal;
+
+/**
+ * Allows autoFlowControl to be turned on and off from interop testing and flow control windows to
+ * be accessed. For internal use only.
+ */
+@VisibleForTesting // Visible for tests in other packages.
+@Internal
+public final class InternalHandlerSettings {
```

Comment:
maybe because I just ran refactor, maybe the file named HandlerSettings.java is still there